### ISSUE TITLE
Windows: app doesn't have publisher, not signed

### ISSUE THREAD

**Issue by slaFFik** [spatie/ray#49]
**Describe the bug**

When trying to install the app shows the screenshot attached below.

**Versions**
Ray version (you can see this in "About Ray"): 1.0.4

**To Reproduce**
1. Make a clean installation of a new version of the app

**Expected behavior**

There should be no security-related notifications from Windows when launching the app.

**Screenshots**

![image](https://user-images.githubusercontent.com/282234/104015323-1c46c180-51bd-11eb-8881-c1fe763f040e.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro, 20H2

**Comment by freekmurze** [comment]
We are waiting on the correct certificates to do this. It's high on our priority list.

**Comment by lovekrissh143** [comment]
Whenever it installs a new version my antivirus squeaks for couple of seconds and then gets back to normal after scanning. 

Maybe the certificates will be corrected so that windows and antivirus will know it's coming from a genuine publisher which we trust more than anything.

I was just using it normally and suddenly the update came and started installing by itself.

And also the check for updates is not having any feedback as it was on earlier versions.

@freekmurze, I think you might know about all of this. :)

**Comment by freekmurze** [comment]
I hope all of this will be fixed when we have the right certificates in place.

**Comment by AngelOD** [comment]
Any updates on this? Would be awesome to be able to use Ray on my work laptop as well.

**Comment by freekmurze** [comment]
We're still waiting on the certificates, it takes a while before these get issued.

When there is news, I'll update this issue.

**Comment by AngelOD** [comment]
Ah okay. I thought it was a fairly quick process to get certificates, but I suppose it makes sense that it takes a while, for security/verification purposes.

**Comment by freekmurze** [comment]
It's a bit frustrating for us too that it takes this long.

**Comment by crypto-scythe** [comment]
Is there any update on this issue?

**Comment by MOuli90** [comment]
I am interested in this Issue. Do you have some new updates?

**Comment by slaFFik** [comment]
@spatie-bot it's not possible to reopen it :)

**Comment by slaFFik** [comment]
It's not possible to reopen it :) There is no such button for regular users.

**Comment by AdrianMrn** [comment]
We've ordered a new key to have another go at this. Hoping to get this one done this or next week.

**Comment by slaFFik** [comment]
![image](https://user-images.githubusercontent.com/282234/168048087-ef33c2fc-37ad-43ae-80af-9f05c312f4d0.png)

**Comment by slaFFik** [comment]
![image](https://user-images.githubusercontent.com/282234/168048087-ef33c2fc-37ad-43ae-80af-9f05c312f4d0.png)

**Comment by slaFFik** [comment]
While waiting for this issue to be resolved I switched to Mac and it's no longer relevant to me :)
